backend/pom.xml

```diff
@@ -24,6 +24,8 @@
 		<java.version>21</java.version>
 		<spring-boot.build-image.imageName>${image.name}</spring-boot.build-image.imageName>
 		<revision>0.0.0-00000000-0000</revision>
+
+		<immutables.version>2.8.8</immutables.version>
 	</properties>
```

Comment:
Consider adding `<m2e.apt.activation>jdt_apt</m2e.apt.activation>` to send a hint to eclipse and vscode that we have an annotation processor configured.

Reply:
added